packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -99,6 +109,18 @@ export const ActionColumn = <T extends Record<string, unknown>>({
           menuItems={headerCheckBoxes}
           onHide={() => setIsOpen(false)}
           onShow={() => setIsOpen(true)}
+          style={{
```

Comment:
You are currently passing `style` and `className` but missing the rest of the `dropdownMenuProps`.

Reply:
Oops, right, looks like I might have deleted that. I included all the other props also and included a test to check if other props are also being included from `ActionColumn` to `DropdownMenu`